pkgs/tools/misc/somafm-cli/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, makeWrapper
+, curl
+, jq
+, mpv
+, rsync
+}:
+
+stdenv.mkDerivation rec {
+  pname = "somafm-cli";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "rockymadden";
+    repo = "somafm-cli";
+    rev = "v${version}";
+    sha256 = "1h5p9qsczgfr450sklh2vkllcpzb7nicbs8ciyvkavh3d7hds0yy";
+  };
+
+  nativeBuildInputs = [ makeWrapper rsync ];
+
+  installPhase = ''
+    make bindir=$out/bin install
+    wrapProgram $out/bin/somafm --prefix PATH ":" ${curl}/bin:${jq}/bin:${mpv}/bin ;
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Listen to SomaFM in your terminal via pure bash";
+    homepage = "https://github.com/rockymadden/somafm-cli";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
I don't have a Darwin box right now, so unable to test, but dependencies are available for Darwin. Is there another reason why it's limited to linux?

```suggestion
```

Reply:
Not that I know of. I mainly added it because I couldn't test it.

Reply:
You need to include `meta.platforms`. You can probably set to `all`.

Reply:
@aanderse I created https://github.com/NixOS/nixpkgs/pull/96643 to address this.

Reply:
@aanderse I have been telling people to remove it when it is `all`, since that is the default ;). I started doing that when some reviewer somewhere told people to do that.

I am fine either way.

Reply:
Oof! I hate giving bad advice so thanks for bringing that up. I think historically people asked for it to always be there so maybe a point to bring up on discourse to raise awareness? Thanks for mentioning to me! And again... sorry for the bad advice!

Reply:
I think it's probably either way, but some builders set it to a specific value if you leave it out, e.g. `buildRustPackage` and `buildGoModule` set it to the platforms supported by the Rust and Go compilers respectively. I guess that's the motivation to leave it out.